packages/docs/src/components/banner.js

```diff
@@ -16,14 +11,14 @@ export default props =>
           fontWeight: 900,
           letterSpacing: '0.05em',
           mt: 0,
-          mb: 4,
+          mb: 4
         },
         p: {
           fontWeight: 'bold',
           mt: 0,
           mb: 3,
           ':first-of-type': {
-            fontSize: [2, 3, 4],
+            fontSize: [2, 3, 4]
```

Comment:
NB: We can enable hanging commas in the prettier setup to avoid diffs like this

Reply:
Prettier is set up now 👍